lib/evaluate/intrinsics-library.cc

```diff
@@ -50,29 +50,53 @@ bool HostIntrinsicProceduresLibrary::HasEquivalentProcedure(
 
 // Map numerical intrinsic to  <cmath>/<complex> functions
 
-// C++ Bessel functions take a floating point as first argument.
+// Define which host runtime functions will be used for folding
+
+// C++ Bessel functions take a floating point as first arguments.
 // Fortran Bessel functions take an integer.
-template<typename HostT> static HostT Bessel_jn(std::int64_t n, HostT x) {
+template<typename HostT> static HostT Bessel_j0(HostT x) {
+  return std::cyl_bessel_j(0., x);
+}
+template<typename HostT> static HostT Bessel_y0(HostT x) {
+  return std::cyl_neumann(0., x);
+}
```

Comment:
When I compile with clang, I get these errors:
```
../../lib/evaluate/intrinsics-library.cc:58:15: error: no member named 'cyl_bessel_j' in namespace 'std'
  return std::cyl_bessel_j(0., x);
         ~~~~~^
../../lib/evaluate/intrinsics-library.cc:61:15: error: no member named 'cyl_neumann' in namespace 'std'
  return std::cyl_neumann(0., x);
         ~~~~~^
```

Reply:
Thanks for testing and catching this. I had only checked clang with libstd++... It seems Libc++ has no support yet of C++17 new math functions.
I changed the code to use C Bessel functions instead. It compiles and pass tests with both GCC, clang + GNU libstd++ and clang + libc++ without warnings.